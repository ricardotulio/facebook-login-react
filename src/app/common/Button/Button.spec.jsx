import React from 'react'
import renderer from 'react-test-renderer'

import Button from './Button'

test('Properties are passed to element', () => {
  const button = renderer.create(
    <Button type="button" onclick="() => {}">Click here</Button>
  )
  let tree = button.toJSON()

  expect(tree).toMatchSnapshot()
})