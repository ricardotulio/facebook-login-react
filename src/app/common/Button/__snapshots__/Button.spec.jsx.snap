// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties are passed to element 1`] = `
<button
  onclick="() => {}"
  type="button"
>
  Click here
</button>
`;
